go/vt/schemadiff/column.go

```diff
@@ -171,6 +170,24 @@ func (c *ColumnDefinitionEntity) ColumnDiff(
 		return nil, nil
 	}
 
+	getEnumValuesMap := func(enumValues []string) map[string]int {
+		m := make(map[string]int)
```

Comment:
Minor, but you can allocate the space once up front:
```
m := make(map[string]int, len(enumValues))
```

Reply:
Done.